openapi/code/package.go

```diff
@@ -350,15 +350,15 @@ func (pkg *Package) Load(spec *openapi.Specification, tag openapi.Tag) error {
 			for _, list := range [][]openapi.Parameter{path.Parameters, op.Parameters} {
 				for _, v := range list {
 					param, err := pkg.resolveParam(&v)
-					if param.In == "header" {
-						continue
-					}
 					if err != nil {
 						return fmt.Errorf("no components found: %s %s", verb, prefix)
```

Comment:
```suggestion
						return fmt.Errorf("could not resolve parameter %s for %s %s. This could be due to a problem in the definition of this parameter. If using $ref, ensure that $ref is used inside the 'schema' keyword.", v.Name, verb, prefix)
```